issue with instructor training completion rate script

@maneesha see the error message below that the instructor training completion rate script is throwing. I can't reproduce locally, so I'm disabling this script from the feed for now.


https://github.com/carpentries/feeds.carpentries.org/runs/1613157921?check_suite_focus=true#step:18:121

```
python3 python/instructor_training_completion_rates.py
Traceback (most recent call last):
  File "/home/runner/.local/lib/python3.9/site-packages/pandas/core/indexes/base.py", line 706, in astype
    casted = self._values.astype(dtype, copy=copy)
ValueError: invalid literal for int() with base 10: '3 (badged)'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/runner/work/***/***/python/instructor_training_completion_rates.py", line 60, in <module>
    ax = completion_rates.plot(kind='bar', title =plot_title, figsize=(8,5), legend=False, fontsize=12)
  File "/home/runner/.local/lib/python3.9/site-packages/pandas/plotting/_core.py", line 955, in __call__
    return plot_backend.plot(data, kind=kind, **kwargs)
  File "/home/runner/.local/lib/python3.9/site-packages/pandas/plotting/_matplotlib/__init__.py", line 61, in plot
    plot_obj.generate()
  File "/home/runner/.local/lib/python3.9/site-packages/pandas/plotting/_matplotlib/core.py", line 280, in generate
    self._make_plot()
  File "/home/runner/.local/lib/python3.9/site-packages/pandas/plotting/_matplotlib/core.py", line 1434, in _make_plot
    self.tick_pos = ax.convert_xunits(self.ax_index).astype(np.int)
  File "/home/runner/.local/lib/python3.9/site-packages/pandas/core/indexes/base.py", line 708, in astype
    raise TypeError(
TypeError: Cannot cast Index to dtype int64
make[1]: *** [Makefile:31: plots] Error 1
make: *** [Makefile:8: everything] Error 2
make[1]: Leaving directory '/home/runner/work/***/***'
```

--- a/python/instructor_training_completion_rates.py
+++ b/python/instructor_training_completion_rates.py
@@ -50,7 +50,6 @@ completion_rates.columns = completion_rates.columns.droplevel()
 
 # Replace count of 3 with count of badged and rename the column.  People sometimes get badged without having all three steps checked.
 completion_rates.loc[3] = tp_pass['Badged'][tp_pass['Badged'] == True].count()
-completion_rates.rename(index={3:'3 (badged)'}, inplace=True)
 
 total_trainees = completion_rates['count'].sum()
 
@@ -65,8 +64,6 @@ for p in ax.patches:
     percentage = int((p.get_height() / completion_rates['count'].sum()) * 100)
     annotation = str(int(p.get_height())) + " (" + str(percentage) + "%)"
     ax.annotate(annotation, (p.get_x(), p.get_height() + 15 ))
-
-ax.set_ylim(0, completion_rates.loc['3 (badged)']['count'] + 100)
 
 plt.xticks(rotation='horizontal')
 

